apps/ccf-body-ui-wc/src/environments/environment.staging.ts

```diff
@@ -14,7 +14,7 @@ export const environment = {
     hubmapPortalUrl: 'https://portal.hubmapconsortium.org/',
     hubmapDataUrl: 'https://search.api.hubmapconsortium.org/v3/entities/search',
     hubmapAssetsUrl: 'https://assets.hubmapconsortium.org',
-    hubmapToken: localStorage.getItem('HUBMAP_TOKEN') ?? '',
+    token: localStorage.getItem('SESSION_TOKEN') ?? '',
```

Comment:
You can remove this kind of usage too. That was a debugging thing used long ago, but no longer relevant.